pkg/api/http.go

```diff
@@ -70,6 +71,7 @@ const (
 	PathPrefixGenerator = "/generator"
 
 	PathTraces              = "/api/traces/{traceID}"
+	PathTraceLookup         = "/api/trace-lookup"
```

Comment:
might need to talk about this. i don't like `/api/trace-lookup` but i can't think of anything off the top of my head i do like.

i'm also wondering if this intended to be used as a POST? since we seem to be reading the body below?